From hackermail Thu Jan  1 00:00:00 1970
From: SeongJae Park <sj@kernel.org>
To: Andrew Morton <akpm@linux-foundation.org>
Cc: SeongJae Park <sj@kernel.org>, Jonathan Corbet <corbet@lwn.net>, damon@lists.linux.dev, kernel-team@meta.com, linux-doc@vger.kernel.org, linux-kernel@vger.kernel.org, linux-mm@kvack.org, Stephen Rothwell <sfr@canb.auug.org.au>
Message-Id: <20250305182744.56125-1-sj@kernel.org>
Date: Wed,  5 Mar 2025 10:27:44 -0800
Subject: [PATCH mm-unstable] Docs/mm/damon/design: fix a typo on 'intervals auto-tuning' section

Commit 8a2e41c7ecfd ("Docs/mm/damon/design: document for intervals
auto-tuning") on mm-unstable tree made a typo on a cross reference link
in monitoring intervals auto-tuning section.  It causes document build
warning.  Fix it.

Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Closes: https://lore.kernel.org/20250305203312.6f30e9c2@canb.auug.org.au
Fixes: 8a2e41c7ecfd ("Docs/mm/damon/design: document for intervals auto-tuning") # mm-unstable
Signed-off-by: SeongJae Park <sj@kernel.org>
Link: https://patch.msgid.link/20250305182744.56125-1-sj@kernel.org
---
 Documentation/mm/damon/design.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/mm/damon/design.rst b/Documentation/mm/damon/design.rst
index 0cf678d98b1b..aae3a691ee69 100644
--- a/Documentation/mm/damon/design.rst
+++ b/Documentation/mm/damon/design.rst
@@ -340,7 +340,7 @@ DAMON-observed access events to the theoretical maximum amount of the events
 (``aggrs``).
 
 The DAMON-observed access events are calculated in byte granularity based on
-DAMON :ref:`region assumption <damon_design_region_based_sample>`.  For
+DAMON :ref:`region assumption <damon_design_region_based_sampling>`.  For
 example, if a region of size ``X`` bytes of ``Y`` ``nr_accesses`` is found, it
 means ``X * Y`` access events are observed by DAMON.  Theoretical maximum
 access events for the region is calculated in same way, but replacing ``Y``

base-commit: b6e1916d98e95c70d059da8803c2b3c46151303c
-- 
2.39.5